### ISSUE TITLE
Issue in endpoint completion

### ISSUE THREAD

**Issue by Shan1024** [ballerinalang/plugin-vscode#20]
When the endpoint is completed, it looks like this.

```ballerina
function test () {
    endpoint <${1:constraint}> ${2:__endpoint} {
        ${3}
    }
}
```

**Comment by roonyh** [comment]
@nadeeshaan Seems to be an issue with language server. Can you take a look?

**Comment by NipunaMarcus** [comment]
Can we close this as this is fixed in the current master?

**Comment by roonyh** [comment]
Fixed through ballerinalang/language-server#51 by @nadeeshaan
